spec/actions_spec.rb

```diff
@@ -137,6 +138,30 @@ def self.name
       expect(another_instance.id).to eq instance.id
       expect(another_instance.value).to eq instance.value
     end
+
+    it '#where' do
+      klass.establish_connection
```

Comment:
Why do we need to call `establish_connection` for this test and the one below?

Reply:
we're resetting the redis database between examples. `establish_connection` would prepare the redis server (loading scripts and model definitions)

Reply:
Ah right, and it fails here because model definitions with index attributes were flushed from redis. Makes sense!